### DISCUSSION TITLE
This isn't an issue but it has to be said:

### DISCUSSION THREAD

**Discussion by nastynaz** [ryan4yin/nixos-and-flakes-book#43] [General]
I just wanted to drop you a note to say thanks for putting together that fantastic guide on NixOS Flakes. As a developer who's been wrestling with config file chaos for years, finding this repo & your guide felt like stumbling onto an oasis.

In less than two days, I was able to consolidate config files I've been accumulating for years into one clean system. This has saved me more hours of frustration than I can count. Honestly, I can't overstate the impact this has had on my workflow and sanity.

I just wanted you to know that your effort and generosity haven't gone unnoticed. Hopefully someday I can pay it forward and build something half as useful for others.

You have my deepest gratitude.
Thank you! 🙏

**Comment by ryan4yin** [comment]
Thanks for your feedback! I'm glad it helped you٩(๑ơలơ)۶♡

**Comment by zyrain** [comment]
I too came to say the same thing.  I was close to throwing in the towel on learning Nix and Flakes, having struggled for weeks on getting my system set up.  The mindset you bring to this, especially the "there's a lot of well intentioned, but old or misleading information out there", is lighting the way through the darkness.  Please keep this up to date!
